packages/qwik-query/package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "@tanstack/qwik-query",
+  "version": "0.0.1",
+  "description": "Primitives for managing, caching and syncing asynchronous and remote data in Qwik",
+  "main": "./lib/index.qwik.mjs",
+  "qwik": "./lib/index.qwik.mjs",
+  "types": "./lib-types/index.d.ts",
+  "exports": {
+    ".": {
+      "import": "./lib/index.qwik.mjs",
+      "require": "./lib/index.qwik.cjs",
+      "types": "./lib-types/index.d.ts"
```

Comment:
as far as I know, `types` must come first in `exports`

Reply:
done